README.md

```diff
@@ -11,6 +11,12 @@ Tested with:
 - SQlite3
 - MySQL/MariaDB
 
+## Installation
+Add this lin to you application's Gemfile:
+`gem 'fork-activesupport-cache-database'`
```

Comment:
could you pls use remote the `fork-`?

Reply:
also, this should be a code block (not inline code), use three backticks pls and leave spaces before and after.